.github/workflows/deploy_to_aws.yml

```
@@ -5,6 +5,9 @@ on:
     branches:
       - main
       - 'releases/**'
+  create:
+    branches-ignore:
```

actionlint (1.7.7)

"branches-ignore" filter is not available for create event. it is only for merge_group, push, pull_request, pull_request_target, workflow_run events

(events)